themes/app/templates/Includes/HomePageAccom.ss

```diff
@@ -0,0 +1,25 @@
+<div class="accommodation container">
+    <div class="row justify-content-center">
+        <h2 class="accommodation__heading">$AccomHeading</h2>
+    </div>
+    <div class="row justify-content-center">
+        <div class="accommodation__stage col-lg-3 col-md-3 col-12" align="center">
+            <img class="accommodation__icon" src="$ThemeDir/dist/images/House.png" class="accom__img" />
+            <a class="accommodation__desc" href="$Accom1Link.Link"><h3>$AccomStep1</h3></a>
+        </div>
+        <div class="col-lg-1 col-md-1 col-12" align="center">
+            <img class="accommodation__arrow" src="$ThemeDir/dist/images/Arrow.png"/>
```

Comment:
I would try to get `.svg` files for your icons so it scales nicely and you don't have to worry about adjusting its width on certain breakpoints :)

Reply:
<img width="578" alt="screen shot 2018-12-06 at 11 19 27 am" src="https://user-images.githubusercontent.com/27745093/49547875-d6b5cb80-f948-11e8-92d9-dd301597871f.png">

Here you can see how our `icon` is bleeding out of the container because of the `width: 40px` applied onto the `icon`. Ideally the use of `svgs` here should resolve the scaling issues :)

Reply:
Building on further from this, I noticed that the position of the arrow is determined by a `padding-top` attribute, I would try avoid this as responsiveness doesn't play too nicely with this.

I would try use `vertical-align: middle` on each of the icons so they all sit nicely in the middle. Could also experiment with alternative solutions..